src/app/zap-templates/zcl/zcl.json

```diff
@@ -135,5 +135,6 @@
             "OverrunCount"
         ]
     },
-    "defaultReportingPolicy": "mandatory"
+    "defaultReportingPolicy": "mandatory",
+    "ZCLDataTypes": ["ARRAY", "BITMAP", "ENUM", "NUMBER", "STRING", "STRUCT"]
```

Comment:
Does NUMBER include floats?

What is this entry used for on the ZAP side?

Reply:
Number includes floats. It includes everything that is not determined as a bitmap, enum, string or struct.

This entry is used to form a discriminator table for determining all the zcl data types which is then used to load all the types defined in the xml.